### ISSUE TITLE
[CAIA Intake] <Benefits Dependents Experience>: <Improve Dependent Card Info>

### ISSUE THREAD

**Issue by steele-lm** [department-of-veterans-affairs/va.gov-team#89314]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to request collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

## About your team

- **Team name**: Benefits - Dependent Experience
- **OCTO product owner:** Ryan Powers
- **Product name:** 21-686c and 21-674 online form flow
- **Product manager:**  Laura Steele
- **Slack channel:** #benefits-dependents-management
- **Dedicated content writer on your team (if you have one):** none
- **Dedicated a11y specialist on your team (if you have one):** none
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [ ] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [x] Something else: Adding new authenticated content to VA.gov

### What's the nature of your initiative and desired outcomes?

When a Veteran currently visits the ["view dependents" page](https://www.va.gov/view-change-dependents/), they are shown a limited amount of information about each of their dependents (see screenshot below). This limited information may not be enough for the Veteran to accurately understand what dependents may need to be updated or the status of their dependents, which can lead to the under/over payment of benefits. This project will add info about the benefit the dependent is on, the status of the dependent, and the planned date of removal from the benefit to the dependent card.

Existing dependent card:
![image](https://github.com/user-attachments/assets/e62cae3a-292a-4445-a274-25045001a351)

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [ ] No
- [x] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

We are working on a broader solution for dependent verification on VA.gov [here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/83353), and updating the info displayed on the dependent cards is a complementary project. We are hoping to launch both the dependent verification solution and the updated dependent cards by March 2024.

### Will you release this new product incrementally (for example, release to 25% of users to start)?

- [ ]  Yes
- [x]  No
 
 **Note:** For an incremental launch, we use React widgets to display dynamic content. We'll need at least 2 weeks of extra time to coordinate the content for this and work with your engineer. [Learn more about our process for dynamic content](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/sitewide-content/processes/dynamic-content-process.md). If your launch plans change, notify CAIA of the change as soon as possible so we can talk about how that will impact the timeline.

**If you are releasing incrementally:**
- **What dates do you anticipate starting and ending your incremental launch?** `Enter dates`
- **What is the name of your engineer who will build the React widget?** `Name`

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [x] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [ ] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** #83353

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Wireframe for updated dependent card](https://www.figma.com/design/bvj72inycD0iZkuCbjYTWL/Dependent-Verification-MVP?node-id=210-12530&t=gqeVTdfqCFtiMbmy-0)

## Next steps
- [x] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Naomi Eke`, and we’ll then acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well: [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work: [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team: [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next: [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).

**Comment by steele-lm** [comment]
@ajialeilani @Dene-human @juliepedtke

**Comment by kristinoletmuskat** [comment]
Thank you @steele-lm ! Which page / frame or area in Figma should we be looking at?

**Comment by steele-lm** [comment]
@kristinoletmuskat I think the link takes you right to it...it's the "enriched card". Thanks!

**Comment by kristinoletmuskat** [comment]
@steele-lm here's a screenshot of what I see when I click the figma link -- a lot going on! Maybe we could draw a box or something around the card(s) you want us to review?

![Screenshot 2024-07-30 at 3 14 57 PM](https://github.com/user-attachments/assets/aa0642ca-5ec6-4d01-bf38-3626e500f8cf)

**Comment by steele-lm** [comment]
@ajialeilani Can you highlight what @kristinoletmuskat should review on the Figma file for the dependent cards?

**Comment by ajialeilani** [comment]
Hey @kristinoletmuskat sorry about the confusion—I created a new page with just the proposed and current cards [here](https://www.figma.com/design/bvj72inycD0iZkuCbjYTWL/Dependent-Verification-MVP?node-id=594-29700&t=BZRyxp9iH9NdyX6I-1). I'll attach an image of the proposed card below as well. 
![Enriched card](https://github.com/user-attachments/assets/81655892-19a3-44b7-81d3-34d05e4fab83)

**Comment by kristinoletmuskat** [comment]
Awesome! This looks great @ajialeilani A few Qs below:

1. Could you clarify when/where we're going to use the enriched card vs the regular card?
2. Are the attributes on the enriched card the full set of attributes we have about dependents? Or are there more user-facing attributes that could show up? Also noting that in the 'verify' flow, it looks like dependent 'status' is not one of the items users are verifying. I'm not sure if that one was left off intentionally (like if it has no impact on benefits) or not!
3. Perhaps for the future we could consider making these object cards [directly manipulatable](https://www.nngroup.com/articles/direct-manipulation/).
-  For example, imagine I'm a user who just got divorced and want to remove my ex-spouse from my benefits (or my dependent passed away and I want to remove them so I don't get overpaid by the VA). These might not be real use cases, but just as an example. I see my spouse listed, but I have to put in some extra thinking to figure out how to edit/remove them by scanning the rest of the page -- the affordance for editing an object is disconnected from the object itself. If there was a little 'edit/delete' icon in the card, I might not have to spend that extra mental energy figuring out how to interact with the object.  

Thank you!

**Comment by kristinoletmuskat** [comment]
Hey @ajialeilani , just checking in on the Q above and wanted to see where this work is at? Do you have everything you need from us?

**Comment by coforma-jamie** [comment]
Hi @ajialeilani, took a look at the card from an accessibility POV and had a couple of thoughts:
- Make sure the **Firstname Lastname** heading is at the right level for where it'll sit on the page. I'm assuming these will be `H3`, where "Your VA dependents" is `H1` and "Dependents on your VA benefits" is `H2`
- Consider putting the attributes in bold for ease of scannability, like:
   - **Relationship:** Child
   - **Status:** Student 
- If you get to the point where you're able to make actions in the card directly manipulable, like Kristin noted above, you'll need to think about links v buttons (is the action taken in-card, or a link to another page?) and accessible names (should the visible edit text be "Edit relationship," should we use aria-labelledby for that?)

Also seconding what Kristin said about dependent attributes (status, relationship, etc) - are there any other attributes that are known about dependents? Would any of those attributes be useful to add to the card? The less hunting and clicking someone has to do, the easier it'll be for a user to understand and take action on.
